eng/targets/Npm.Common.targets

```diff
@@ -76,7 +76,7 @@
     DependsOnTargets="GetBuildInputCacheFile"
     Inputs="@(TSFiles);$(BaseIntermediateOutputPath)tsfiles.cache"
     Outputs="@(BuildOutputFiles)">
-    <Yarn Command="$(NpmBuildArgs)" StandardOutputImportance="High" StandardErrorImportance="High" IgnoreStandardErrorWarningFormat="$(IgnoreYarnWarnings)" />
+    <Yarn Command="$(NpmBuildArgs) --openssl-legacy-provider" StandardOutputImportance="High" StandardErrorImportance="High" IgnoreStandardErrorWarningFormat="$(IgnoreYarnWarnings)" />
```

Comment:
Use my approach instead unless this is necessary for some job other than the Windows ARM64 one. Just wasteful to build Node JS projects there. Only got into this state because the images were updated.